Dockerfile

```diff
@@ -33,7 +33,8 @@ RUN set -x \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
 RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
+	mkdir /tmp/composer && \
+    mv composer.phar /tmp/composer/composer
```

Comment:
This does not solve the problem with **"once you have used `sudo make composer_install` and you have lost the composer binary, you are no longer able to install composer via `make composer_download`"** (unless you use admin privileges remove `~/.composer`) see [comment above](https://github.com/fossology/fossology/pull/731#discussion_r96665109).

What speaks against using the `$PATH` variable to tell the makefiles where to look for composer?